### DISCUSSION TITLE
is there any plan for an integration w/ a headless cms to edit content ?

### DISCUSSION THREAD

**Discussion by M3lvin** [thuliteio/doks#1232] [Q&A] [answered]
Hello, 

I am interested in using doks to build a specific wiki, it looks amazing. 
I saw this specific article on ventizo https://ventizo.com/products/anna/ and so it seems there was a plan to peer a headless cms (forestry at the time) w/ this template but I'm guessing this is not up-to-date anymore (talking about 2022, forestry etc). 

So I was wondering if there was any other plan to integrate it w/ any other headless cms, like contentful or others ? 
The objective : avoid the markdown editing for non-technical people and collaborate more easily. 

I will look into the integration config (w/ contentful) during my next vacation but wanted to know if anyone already digged into this. 

Thanks!

**Comment by h-enk** [comment, marked as answer]
A while ago, I set up a Proof of Concept to see how Doks could work with [Tina](https://tina.io/) (and Decap). I noticed that if you do not specify the content structure completely (one on one), frontmatter gets overwritten (I did not like that — but I get it). I also remember that page bundle images are not supported (in Tina). And I know that Doks is used in many different ways, making it hard/impossible to come up with a generic setup that works for everyone. It is still on my mind, but I'm not sure...

See also [Use a CMS with Hyas](https://docs.gethyas.com/guides/cms/) with (general) guides for Tina, Decap, and CloudCannon

I also stumbled upon [Contentful Hugo](https://github.com/ModiiMedia/contentful-hugo) — maybe interesting to have a look at

**Reply by M3lvin** [reply]
Thanks, this looks like a good way to start ;).
